packages/template-retail-react-app/app/components/store-locator-modal/index.jsx

```diff
@@ -26,7 +29,20 @@ import {
 } from '@salesforce/retail-react-app/app/constants'
 
 export const StoreLocatorContext = createContext()
+export const StoreLocatorProvider = ({children}) => {
+    const storeLocator = useStoreLocator()
+
+    return (
+        <StoreLocatorContext.Provider value={storeLocator}>{children}</StoreLocatorContext.Provider>
+    )
+}
+
+StoreLocatorProvider.propTypes = {
+    children: PropTypes.node.isRequired
+}
```

Comment:
would it be preferred to move this to the [contexts/index.js](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/app/contexts/index.js)?